### DISCUSSION TITLE
Active and Static leaves to public.

### DISCUSSION THREAD

**Discussion by ghost** [bepu/bepuphysics2#225] [Q&A]
Currently in the latest version of V2.3 active and static leaves are public. I want to upgrade to .NET 6 and want to use the latest version "V2.4". Is it possible to make the active and static leaves within the BroadPhase class public again?

**Comment by RossNordby** [comment]
They are in latest master. The 2.5.0-beta.X versions include (at least relatively) recent snapshots.

**Reply by ghost** [reply]
Thank you for the quick response and help!
